### PR TITLE
Release v2.2

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: _data/versions.yml [hunk 1]
```diff
@@ -1,3 +1,183 @@
+v2.2:
+- title: v2.2.0
+  note: |
+    10-May 2017
```
Comment (caseydavenport) [inline _data/versions.yml]: This needs updating.

File: _data/versions.yml [hunk 2]
```diff
@@ -1,3 +1,183 @@
...
+    
+    This release includes new versions most most of the key Calico components. Highlights include
```
Comment (caseydavenport) [inline _data/versions.yml]: "most most"

File: _data/versions.yml [hunk 3]
```diff
@@ -1,3 +1,183 @@
...
+    calicoctl:
+      version: v1.2.0
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.2.0
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.2.0/calicoctl
+    calico/node:
+      version: v1.2.0
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.2.0
+    calico/cni:
+      version: v1.8.1
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v1.8.1
```
Comment (caseydavenport) [inline _data/versions.yml]: I think this should be v1.8.3 now.

File: _data/versions.yml [hunk 4]
```diff
@@ -1,3 +1,183 @@
...
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.8.1/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.8.1/calico-ipam
+    calico-bird:
+      version: v0.3.0
+      url: https://github.com/projectcalico/calico-bird/releases/tag/v0.3.0
+    calico-bgp-daemon:
+      version: v0.2.1
+      url: https://github.com/projectcalico/calico-bgp-daemon/releases/tag/v0.2.0
```
Comment (caseydavenport) [inline _data/versions.yml]: URL needs updating to v0.2.1

**Comment by caseydavenport** [comment]
Also need to resolve conflicts.

File: _data/versions.yml [hunk 5]
```diff
@@ -1,3 +1,257 @@
...
+    This release includes new versions of most of the key Calico components. Highlights include
+    * New version of Bird for BGP multipath support
+    * New version of GoBGP integration with IPIP support and other fixes
+    * A new CNI plugin, supporting the latest version of the CNI spec
+    * Further improvements to the Kubernetes datastore driver, bringing beta support for BGP routing.
+    * Some significant improvements to Felix performance and occupancy.
+
+    For more details see the [blog post](https://www.projectcalico.org/project-calico-2-2-released/) or the component release notes linked below.
+
+    There are two known issues with this release (that also affect previous releases). These issues will be addresses in v2.2.1 to be released shortly.
```
Comment (caseydavenport) [inline _data/versions.yml]: typo: "addresses"

File: _data/versions.yml [hunk 6]
```diff
@@ -1,3 +1,257 @@
...
+    * New version of Bird for BGP multipath support
+    * New version of GoBGP integration with IPIP support and other fixes
+    * A new CNI plugin, supporting the latest version of the CNI spec
+    * Further improvements to the Kubernetes datastore driver, bringing beta support for BGP routing.
+    * Some significant improvements to Felix performance and occupancy.
+
+    For more details see the [blog post](https://www.projectcalico.org/project-calico-2-2-released/) or the component release notes linked below.
+
+    There are two known issues with this release (that also affect previous releases). These issues will be addressed in v2.2.1 to be released shortly.
+    1) Some users may experience route flapping when using IPIP - For more details see https://github.com/projectcalico/bird/pull/37
```
Comment (tmjd) [inline _data/versions.yml]: I think the two issue links should be 'links'.

File: _data/versions.yml [hunk 7]
```diff
@@ -1,3 +1,257 @@
...
+    * New version of Bird for BGP multipath support
+    * New version of GoBGP integration with IPIP support and other fixes
+    * A new CNI plugin, supporting the latest version of the CNI spec
+    * Further improvements to the Kubernetes datastore driver, bringing beta support for BGP routing.
+    * Some significant improvements to Felix performance and occupancy.
+
+    For more details see the [blog post](https://www.projectcalico.org/project-calico-2-2-released/) or the component release notes linked below.
+
+    There are two known issues with this release (that also affect previous releases). These issues will be addressed in v2.2.1 to be released shortly.
+    1. Some users may experience route flapping when using IPIP - For [more details](https://github.com/projectcalico/bird/pull/37)
```
Comment (robbrockbank) [inline _data/versions.yml]: Should this read `See [issue](https://github.com/projectcalico/bird/pull/37) for more details`

Same below.

**Pull request merged**